app/js/controllers/foyer/ressources/ressources.js

```diff
@@ -67,53 +63,23 @@ angular.module('ddsApp').controller('FoyerRessourcesCtrl', function($scope, $sta
                 onGoing: true
             };
 
+            // For autres revenus TNS, we also need to find the CA
+            if (type == 'autresRevenusTns') {
+                ressource.caAnnuel = _.chain(ressources)
+                    .where({ type: 'caAutresRevenusTns' })
+                    .pluck('montant')
+                    .reduce(function(sum, num) {
```

Comment:
`num` is badly named, as it is ambiguous with an index. Use either `montant` for consistency with the `pluck` if you consider it a part of the domain, or `amount` otherwise.